src/ClientProtos/capture_data.proto

```diff
@@ -154,6 +155,8 @@ message TimerInfo {
     kSystemMemoryUsage = 8;
     kCGroupAndProcessMemoryUsage = 9;
     kPageFaults = 10;
+    kApiScope = 11;
```

Comment:
I don't remember: What's would be the difference with `kApiEvent`?

Reply:
`kApiEvent`s have the `ApiEvent` encoded in the registers, they further have another type field in the encoded event. The complete type will go away in the future.
`kApiScope` and `kApiScopeAsync` have their information all explicitly here. No encoding.

Reply:
Ah right. So I'm wondering the following: would it be possible to get rid of `kApiEvent` and only generate `kApiScope`s and `kApiScopeAsync`s now? I know it's a bit more work here but it would clean up at least something here.

Reply:
So I guess the scope timers and the track events could be handled in #2634. I am not sure if we want to merge those PRs into one, though... 

Note that the string events can't be really handled that easy. The producer splits longer strings into a sequence of events, while the new ones will only allow one string event per id.

In general, I like the idea of not producing the old client protos ASAP, that is even before we have the converter. However, unless you strongly object, I would still do this step by step.
1. Introduce new events.
2. Process new events.
3. Convert old events into new events.
4. Remove old events.

Reply:
Sounds all good.